packages/p2p-media-loader-core/src/utils/peer.ts

```diff
@@ -32,3 +34,9 @@ export function generatePeerId(): {
 
   return { string: peerId, bytes: utf8ToUintArray(peerId) };
 }
+
+function formatVersion(versionString: string) {
```

Comment:
Can it be something smarter because this approach just removes `.` from the version string and it means that `1.10.1` will give the same result as `11.0.1`